runtime/onert/backend/acl_common/AclTensorRegistry.h

```diff
@@ -43,10 +46,14 @@ template <typename T_AclTensorManager> class AclTensorRegistry : public ITensorR
     return getITensor(ind);
   }
 
+  auto getAclTensor(const ir::OperandIndex &ind) { return _tensor_mgr->at(ind); }
```

Comment:
I wonder the reason of introducing this method instead of using `getITensor()`. Could you explain a bit? Thanks.

Reply:
It returns the actual implementation of `ITensor`(`CLTensor` or `NETensor`). If it was a raw pointer we can just use [Covariant Return Type](https://www.lwithers.me.uk/articles/covariant.html), but it is not possible with `shared_ptr`.

Plus, I made the return type `auto` otherwise the template code will get more complicated.